code/game/objects/structures/beds_chairs/bed.dm

```diff
@@ -265,4 +273,4 @@
 /obj/structure/bed/double/alien/examine(mob/user)
 	. = ..()
 	if(isabductor(user))
-		. += "<span class='abductor'>Fairly sure we absoluteley steal that technology...Why did we steal this again?</span>"
+		. += "<span class='abductor'>Fairly sure we absoluteley stole that technology...Why did we stole this again?</span>"
```

Comment:
Copy paste this please.
`Fairly sure we absolutely stole that technology... Why did we steal this again?`
`absoluteley` is wrong, second one is `steal`